.github/workflows/compress-images.yml

```diff
@@ -0,0 +1,31 @@
+name: Compress Images
+on:
+  pull_request:
+    # Run Image Actions when JPG, JPEG, PNG or WebP files are added or changed.
+    # See https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#onpushpull_requestpaths for reference.
+    paths:
+      - '**.jpg'
```

Comment:
also add the workflow file to ensure it is run when it changes. We always do this with workflows

Reply:
done